### PR TITLE
MERN Crud

### PR DESCRIPTION
**Pull request by nisalrenuja**

### CODE REVIEW

**Comment by blueseaturtle** [comment]
Thanks for your contribution!

Would you be able to add a small README file to the root of your `MERN_CRUD` folder describing what the project does, and how both the server and client can be run locally? It'll be very helpful.

Please let me know if you'll be able to ✌️

**Comment by nisalrenuja** [comment]
sure I'll

**Review by blueseaturtle — CHANGES_REQUESTED** [review]
The commands needed to set up and run the project seem to be missing.
Something like [this](https://github.com/blueseaturtle/The-CRUD-Collection/tree/main/React_Redis/RhymePingPong#local-installation) would be great.

File: MERN/MERN_CRUD/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+3. Run the following command to start the application.
+
+   ```bash
+   npm start
+   ```
+
+4. Open a browser and navigate to `http://localhost:3000/`.
+
+5. Backend
```
Comment (blueseaturtle) [inline MERN/MERN_CRUD/README.md:36]: Frontend

File: MERN/MERN_CRUD/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+- Open a terminal in the root of the project folder.
+- Run cd server to enter the folder.
```
Comment (blueseaturtle) [inline MERN/MERN_CRUD/README.md:39]: Run cd client .....

File: MERN/MERN_CRUD/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- Run npm install.
+
```
Comment (blueseaturtle) [inline MERN/MERN_CRUD/README.md:41]: Run npm start

File: MERN/MERN_CRUD/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+React
+Node
+
+## Prerequisites
+
+- Add MongoDb to this MERN app. [View documentation here.](https://docs.mongodb.com/manual/installation/)
+- Add Express to this MERN app. [View documentation here.](https://expressjs.com/en/starter/installing.html)
+- Add React to this MERN app. [View documentation here.](https://reactjs.org/docs/getting-started.html)
+- Add Node to this MERN app. [View documentation here.](https://nodejs.org/en/download/)
+
```
Comment (blueseaturtle) [inline MERN/MERN_CRUD/README.md:18]: Add MongoDB Connection [Instructions](https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster/)

File: MERN/MERN_CRUD/server.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+app.use(bodyParser.json());
+app.use(cors());
+
+//route middleware
+app.use(postRoutes);
+
+const PORT = 8000;
+
+const DB_URL =
+  "mongodb+srv://nisal:nisal@mernapp.wopdx.mongodb.net/mernCrud?retryWrites=true&w=majority";
```
Comment (blueseaturtle) [inline MERN/MERN_CRUD/server.js:21]: Replace connection string with a place holder.
The use of a .env file would be good too I think, but will need instructions on how to set it up.

**Review by blueseaturtle — CHANGES_REQUESTED** [review]

**Pull request closed without merging**